### ISSUE TITLE
Docker won't start after yum update

### ISSUE THREAD

**Issue by pirish** [docker/docker#22456]
We performed patching over the weekend which had completely broken our docker environment.  All servers are on rhel 7.2 running in a vm on esxi 5.5 . I've tried following some of  the steps from this 
[issue](https://github.com/docker/docker/issues/12087) , but have been unsuccessful. 

**Output of `docker version`:**

```
[root@dockerhost1 ~]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.2 (Maipo)

[root@dockerhost1 ~]# docker -v
Docker version 1.9.1, build 78ee77d/1.9.1
```

I ran the startup command with the -D flag added, output is [here](http://pastebin.com/ExKaMTPa) . Any help would be appreciated

**Comment by justincormack** [comment]
You have not provided enough information to help. It might be related to https://github.com/docker/docker/issues/16902 perhaps. If you do not care about the existing containers you could remove `/var/lib/docker` and restart and then recreate the containers.

**Comment by pirish** [comment]
I'm not sure what information would help.  I provided what logs I had in the pastebin link in my initial request.  I've tried removing /var/lib/docker and recreating it, still has the same errors.  I've also removed /var/lib/docker and tried reinstalling docker to no avail.  Of the 3 hosts, one machine uses the loopback driver, the other two use devicemapper.  I've also tried disabling selinux to see if maybe that was interfering. 
  The issue you mentioned does seem pretty similar.  When docker fails to start, the shm and mqueue mounts are still present.  Unmounting them manually does not throw errors but also doesn't allow docker to start.
Some of the commands outlined in the issue request template can't be run because docker isn't running Let me know what other information you would like or would be helpful.  I appreciated you getting back to me
